rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -835,6 +835,7 @@ protected NodeRunner buildNodeRunner() {
                     getRepositoryLocator()
             ));
         }
+        SignatureService.initialize(getRskSystemProperties());
```

Comment:
something to ponder: command line and all that jazz, do they start the nodeRunner? if not, maybe there is a better place to initialize a singleton.

Reply:
Yep, i will need some help with that. Dont know where it should be the "best" place. 
Maybe @ajlopez could help me?

Reply:
Check federator code. I'm not sure NodeRunner is unique, maybe they have another Start code